db/migrate/20220908105721_add_dp_to_declarations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddDpToDeclarations < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :participant_declarations, :delivery_partner, index: { algorithm: :concurrently }
```

Comment:
Might need to also add a ` null: true`?

Reply:
apparently it already defaults to true. see https://api.rubyonrails.org/v6.1.6.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference